docs/introduction/introduction.md

```diff
@@ -1,68 +1,90 @@
+
+### Configurations and Rules are Defined by a Web Application Siembol UI
+
+- All configurations are stored in JSON format and edited by web forms in order to avoid mistakes and speed up creation and learning time.
+- Configurations are stored in Git repositories.
+- Supporting high integrity use cases with protected GitHub main branches for deploying configurations.
+- Supporting validation and testing configurations. Moreover, Siembol UI supports creating and evaluating test cases.
+- Siembol prefers a declarative JSON language rather than a scripting language like [Stellar](https://stellar.tech/). We consider declarative language with testing and validation to be less error prone and simpler to understand.
+- Supporting OAUTH2/OIDC for authentication and authorization in Siembol UI.
+- All Siembol services can have multiple instances with authorization based on OIDC group membership. This allows multi-tenancy usage without the need to deploy multiple instances of Siembol.
+- We are planning to test and tune OAUTH/OIDC integration with popular identity providers.
+
+### Easy Installation to Try Out with Prepared Docker Images and Helm Charts
+
+- Siembol supports deployment on external [Hadoop](https://hadoop.apache.org/) clusters to ensure high performance. However, we are providing k8s Helm charts for all deployment dependencies in order to test Siembol in development environments.
 
-- Siembol supports deployment on external hadoop clusters to ensure the high performance which we expect at GR. However, we are providing k8s helm charts for all deployment dependencies in order to test siembol in development environments.
 
 Use-Cases
 ---------
 
-### SIEM log collection using open source technologies
+### SIEM Log Collection Using Open Source Technologies
+
+- Siembol can be used to centralize both security data collecting and the monitoring of logs from different sources. In the process of collecting and inspecting logs from third party tools, the format of these logs can vary. Therefore, it is important for Siembol to support the normalization of logs into a standardized format with common fields, such as timestamp. It is often useful to enrich a log with metadata provided by CMDB or other internal systems which are important for building detections. 
+
+- For example, data repositories can be enriched by data classification, network devices by a network zone, username by active directory group, etc. By using Siembol alerting services, CSIRT teams can use the tool to add detection on top of normalized logs. Alerts triggered from the detections are integrated into incident response and defined and evaluated by the Siembol response service. This allows for integration of Siembol with systems such as [Jira](https://www.atlassian.com/software/jira), [Hive](https://hive.apache.org/), or [Cortex](https://www.cortex-ia.com/our-software/), and provides additional enrichments by searching ELK or doing LDAP queries. 
```

Comment:
links to the hive and cortex are wrong

Reply:
Links to TheHive and Cortex are corrected.